### ISSUE TITLE
update functionality uses "at" command; but this is not installed by default on 16.04 systems

### ISSUE THREAD

**Issue by dejonghb** [openvstorage/framework#1124]
/usr/bin/ovs internally uses "at -f /tmp/update now" to get an updat rolling, but the at command is not installed by default on 16.04 systems.

```
root@ftcmp02:/var/log/ovs# at 
The program 'at' is currently not installed. You can install it by typing:
apt install at
```

As "at" is used, the "at" package is a dependency and should be included as such

**Comment by khenderick** [comment]
Odd, which ISO did you use? My installation from a few days back has `at` installed by default.

```
root@ovs100:~# at 10:00
warning: commands will be executed using /bin/sh
at> echo "foo"
at> <EOT>
job 2 at Sat Nov  5 10:00:00 2016
```

```
root@ovs100:~# uname -a
Linux ovs100 4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
root@ovs100:~# lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.1 LTS
Release:    16.04
Codename:   xenial
root@ovs100:~# apt-cache policy at
at:
  Installed: 3.1.18-2ubuntu1
  Candidate: 3.1.18-2ubuntu1
  Version table:
 *** 3.1.18-2ubuntu1 500
        500 http://be.archive.ubuntu.com/ubuntu xenial/main amd64 Packages
        100 /var/lib/dpkg/status
root@ovs100:~# apt-cache rdepends --installed at
at
Reverse Depends:
    at:i386
    at:i386
    at:i386
  ubuntu-server
    at:i386
```

**Comment by khenderick** [comment]
@dejonghb, can you take a look at the above information on your node? I wonder where the difference would be.

**Comment by dejonghb** [comment]
The nodes were installed via PXE & network install (no ISO install)

```
# uname -a
Linux ftcmp02 4.4.0-45-generic #66-Ubuntu SMP Wed Oct 19 14:12:37 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
```

```
# lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.1 LTS
Release:    16.04
Codename:   xenial
```

```
# apt-cache policy at
at:
  Installed: 3.1.18-2ubuntu1
  Candidate: 3.1.18-2ubuntu1
  Version table:
 *** 3.1.18-2ubuntu1 500
        500 http://be.archive.ubuntu.com/ubuntu xenial/main amd64 Packages
        100 /var/lib/dpkg/status
```

```
# apt-cache rdepends --installed at
at
Reverse Depends:
  devscripts
    at:i386
    at:i386
    at:i386
    at:i386
```

Note that _ubuntu-server_ is not installed....

```
# dpkg -l *ubuntu-server*
dpkg-query: no packages found matching *ubuntu-server*
```

**Comment by khenderick** [comment]
Fixed by #1129, packaged in openvstorage-2.7.5-rev.4269.07eb240

**Comment by pploegaert** [comment]
PASSED:
root@e192-1:~# apt-cache rdepends --installed at
at
Reverse Depends:
  openvstorage-core
    at:i386
    at:i386
  devscripts
    at:i386
    at:i386
    at:i386
    at:i386
    at:i386
  devscripts
    at:i386
